config.defaults.yaml

```diff
@@ -55,6 +55,10 @@ docs:
       coordinates: [270.4229, 7.1722]
       period: 0.26901030387060865
       title: W Uma
+    beta_lyr:
+      coordinates: [259.5077971, 22.7978128]
+      period: 0.03979692049952155
```

Comment:
This object has an unusually short period. I suggest including another star with a more typical period (~0.5-a few days).

Reply:
@przemekmroz updated!